script/client

```diff
@@ -49,8 +49,8 @@ Output YAML instead of Perl structures.
 
 Send Accept header to explicitly tell the API what format is accepted
 
-Note that currently only one endpoint (job_templates_scheduling) is able
-to return YAML, and it's deprecated.
+Returning YAML is deprecated, and currently only one endpoint
```

Comment:
is really deprecated ? Only one endpoint works with YAML but I don't think we going this to change into JSON

Reply:
I also still don't really understand the message. Put yourself into the shoes of a newbie wanting to use this. How should they read it?